mpsl/CHANGELOG.rst

```diff
@@ -72,26 +73,27 @@ Added
 
 Changes
 =======
+
 * Removed ``MPSL_RADIO_NOTIFICATION_DISTANCE_425US`` and replaced it by ``MPSL_RADIO_NOTIFICATION_DISTANCE_420US``.
 * On nRF53, the fix for Errata 16 is now applied.
 * The scheduling overhead of a timeslot event is reduced.
 
 Bugfixes
 ========
 
-* Fixed an issue on nRF53 where an assert could occur when using a timeslot.
+Fixed an issue on nRF53 where an assert could occur when using a timeslot.
 
 nRF Connect SDK v1.2.0
 **********************
 
 Added
 =====
 
-* Added a library version with preliminary support for the nRF5340 device.
-  The feature set is the same as in the MPSL library for nRF52.
-  The following library has been added:
+Added a library version with preliminary support for the nRF5340 device.
+The feature set is the same as in the MPSL library for nRF52.
+The following library has been added:
 
-  * :file:`soft-float/libmpsl.a`
+:file:`soft-float/libmpsl.a`
```

Comment:
Why not mention this library name in line 92?

Reply:
We should not change old release notes.